internal/cmd/config.go

```diff
@@ -186,17 +210,27 @@ func readEnvConfig(envMap map[string]string) (Config, error) {
 // - set some defaults if they weren't previously specified
 // TODO: add better validation, more explicit default values and improve consistency between formats
 // TODO: accumulate all errors and differentiate between the layers?
-func getConsolidatedConfig(gs *state.GlobalState, cliConf Config, runnerOpts lib.Options) (conf Config, err error) {
-	fileConf, err := readDiskConfig(gs)
-	if err != nil {
-		return conf, errext.WithExitCodeIfNone(err, exitcodes.InvalidConfig)
+func getConsolidatedConfig(gs *state.GlobalState, cliConf Config, runnerOpts lib.Options) (Config, error) {
+	fileConf, err := readLegacyDiskConfig(gs)
+	if errors.Is(err, fs.ErrNotExist) { //nolint:gocritic
+		fileConf, err = readDiskConfig(gs)
+		if err != nil {
+			return Config{}, errext.WithExitCodeIfNone(err, exitcodes.InvalidConfig)
+		}
+	} else if err != nil {
+		return Config{}, errext.WithExitCodeIfNone(err, exitcodes.InvalidConfig)
+	} else {
+		gs.Logger.Warn("The configuration file has been found on the old path. " +
+			"Please, run again `k6 cloud login` or `k6 login` commands to migrate to the new path. " +
```

Comment:
```suggestion
			"Please, run again `k6 cloud login` commands to migrate to the new path. " +
```

As `k6 login` has been deprecated, I think it's better to not mention it. Right? 🤔

Reply:
If a user has a config file because it used `k6 login influxdb` but they don't have the Cloud they won't be able to migrate anymore. Do you think we should not support them?

Reply:
Mmm.. not sure. Let's leave it for now, then. Although I guess that `k6 login influxdb` will eventually disappear.